apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -155,7 +157,7 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
     }), [ApartmentUnitTypeValue, CommercialUnitTypeValue, FlatUnitTypeValue, ParkingUnitTypeValue, WarehouseUnitTypeValue])
 
     const meterReadingNormalizer: RowNormalizer = async (row) => {
-        const addons = { address: null, unitType: null, propertyId: null, propertyMap: null, meterId: null, meterResourceId: null, readingSubmissionDate: null, invalidReadingSubmissionDate: null, valuesAmount: 0 }
+        const addons = { address: null, unitType: null, propertyId: null, propertyMap: null, meterId: null, meterResourceId: null, readingSubmissionDate: null, invalidReadingSubmissionDate: null, valuesAmount: 0, value1: undefined, value2: undefined, value3: undefined, value4: undefined }
```

Comment:
Why undefined if everything is null here?
And why not move it below ` ] = map(row, 'value')` line, so you can create initial values in place

Reply:
Fixed